src/main/java/practice/StreamPractice.java

```diff
@@ -75,6 +109,11 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * parametrized with Candidate in CandidateValidator.
      */
     public List<String> validateCandidates(List<Candidate> candidates) {
-        return Collections.emptyList();
+        return candidates
+                .stream()
+                .filter(new CandidateValidator())
+                .map(c -> c.getName())
```

Comment:
```suggestion
	CandidateValidator candidateValidator = new CandidateValidator()
        return candidates
                .stream()
                .filter(candidateValidator)
                .map(c -> c.getName())
```
я б навіть так зробив, щоб не плодити об'єкти